stubs/emoji/emoji/core.pyi

```diff
@@ -1,17 +1,38 @@
+from collections.abc import Callable
 from typing import Pattern
 from typing_extensions import Literal
 
 _DEFAULT_DELIMITER: str
 
+class _DeprecatedParameter: ...
+
 def emojize(
     string: str,
-    use_aliases: bool = ...,
+    use_aliases: bool | type[_DeprecatedParameter] = ...,
     delimiters: tuple[str, str] = ...,
     variant: Literal["text_type", "emoji_type", None] = ...,
     language: str = ...,
+    version: float | int | None = ...,
```

Comment:
PEP 484 specifies that type checkers should treat `int` as an implicit subtype of `float`, so `int` is redundant in a union with `float`

```suggestion
    version: float | None = ...,
```